api/services/execution/execution.proto

```diff
@@ -106,3 +110,16 @@ message PauseRequest {
 message ResumeRequest {
 	string id = 1 [(gogoproto.customname) = "ID"];
 }
+
+message PsRequest {
+	string id = 1 [(gogoproto.customname) = "ID"];
+}
+
+message Ps {
```

Comment:
And this gets renamed to `Process` ? I think we already have `Process` so i'm not sure how well they are in sync